### ISSUE TITLE
Get a laser fume filtration system

### ISSUE THREAD

**Issue by johnmckerrell** [DoESLiverpool/somebody-should#1697]
As mentioned in #1696 we've had complaints about the fumes from Sheila. We're thinking of getting a laser fume filter that would alleviate these problems. It's worth considering that it might be useful at some point to run multiple lasers into it so should consider this when picking our option. HPC have some, including https://hpclaser.co.uk/p/bofa-ad-access-laser-fume-filter/ although we still need to work out which is best.

As it happens we came across a fund from Arnold Clark that offers up to £1000 to fund community projects so we're intending to apply to this as a way to afford the filter. https://www.arnoldclark.com/community-fund/community-group/

/cc @DoESLiverpool/laser-maintenance

**Comment by amcewen** [comment]
Cambridge Makespace have one — I think it's [this one](https://hpclaser.co.uk/p/bofa-ad-oracle-laser-fume-filter/) which is £1500+vat, to go with a laser a bit smaller than Gerald.

**Comment by Sean-anotherone** [comment]
Not to poo poo the idea, which is clearly a sensible one, but do check the consumables prices and lifetime before committing to any particular filter.  Moving the machine to the laser room might be a better idea, as no-one seems to complain about that one since the duct went in.

**Comment by Robotorium** [comment]
True, but the laser room isn't super accessible due to room size and how
full it already is. Plus having a lasercutter in the (rentable) events room
is a bonus.

Op wo 18 mei 2022 15:13 schreef Sean Jarman ***@***.***>:

**Comment by amcewen** [comment]
We might want to add some restrictions on what can be cut on machines with a filter — Cambridge Makespace don't allow MDF, for example, because it clogs the filter too easily, but folk who need to cut that can use the other lasers...

**Comment by johnmckerrell** [comment]
I have now applied to the fund requesting £1000 so will report back how that goes.

**Comment by johnmckerrell** [comment]
Apparently I forgot to report it here but this money was paid to us back in July last year. Not that they're chasing but has any progress been made on deciding what to get and procuring it?

**Comment by JackiePease** [comment]
Some filter prices from [AES Solutions](https://www.shopaessolutions.co.uk/dust-fume-extraction-accessories-filters/filters/filters-by-manufacturer/bofa-filters.html)
DeepPleat Pre Filter: £81.43 inc. VAT
Combined Filter for AD Access: £243.52 inc. VAT
It looks like you need both

**Comment by JackiePease** [comment]
@Sean-anotherone is going to take a look at this over the weekend and try to identify the best approach

**Comment by Sean-anotherone** [comment]
I have now spent a bit of time looking at the various options and the conclusion is quite disappointing.....
There is very little information about actual "real" performance.  There are a number of posts suggesting that they don't perform well for very long, which matches my experience of recirculating lab fume hoods.
Given then, that they will be experiencing a demand to remove sticky plywood and acrylic residues, I can only suggest that the cheapest model is the only one to pursue, and even then on the assumption that the filter lifetime is only a matter of hours of actual use.
There is not in my view a "good" option, but I suggest this approach is the least-worst.
Currently have so many links open I've lost track, but there was a £1200 one that seemed to be about the right size for a small laser, I got there from one of the links Jackie originally sent.
Sean


    On Thursday, 25 May 2023 at 18:22:35 BST, Jackie Pease ***@***.***> wrote:  
 
 


@Sean-anotherone is going to take a look at this over the weekend and try to identify the best approach

**Comment by JackiePease** [comment]
I spoke to a sales person at HPC wrt a filter system for use with Sheila. He suggested one of these two: the [BOFA AD Access Laser Fume Filter](https://hpclaser.co.uk/p/bofa-ad-access-laser-fume-filter/) (currently  £923.47 ex. VAT) or the [BOFA AD Oracle Laser Fume Filter](https://hpclaser.co.uk/p/bofa-ad-oracle-laser-fume-filter/) (currently £1,738.25 ex. VAT).
He confirmed that both prefilters and filters are consumables, but wasn't able to give much indication of how long they would last, as this is dependent on materials, settings etc. He did mention that they often lasted for a year in school environments, after which they should be changed anyway.
You can buy a sensor that measures levels of different gases - this could be useful if they don't have noticeable smells, but they can be difficult to use.

**Comment by JackiePease** [comment]
The sensor would be £250.
Pre Filter AD Access £75.86 ex. VAT
Pre Filter AD Oracle £188.37 ex. VAT (or £245.14 ex. VAT for a special one for MDF)
Combined Filter AD Access £240.66 ex. VAT
Combined Filter AD Oracle ££302.00 ex. VAT
[From the spare parts page](https://hpclaser.co.uk/c/spare-parts/)

**Comment by JackiePease** [comment]
We're planning to get the AD Oracle as this appears best for Sheila (or Sophia if this is brought back into use)

**Comment by magman2112** [comment]
I’ve mentioned before that I found a used extractor on Facebook Marketplace, but @JackiePease reminded me last night that I hadn’t posted about it on here.

https://www.facebook.com/marketplace/item/161793030163741/

Although this is listed as a Bofa Ta40ts it is actually the AD400TS/1, which is a more easily identifiable model. The device is for sale for £350 and needs picking up from Wrexham, which I could probably do.

It is stated as having filters at 75%, but whether this is used or time remaining is unclear, but as they were installed in 2020, I think it is safe to assume that these will need replacing.

There appear to be 2 filters in these devices typically, a pre-filter and a combined filter. I’ve found both filters at a few sites, but this one looks quite reasonable.

https://www.totalextraction.co.uk/shop/BOFA_Pre-Filter_A1030045

Pre filters are £191.40 and the combined filter is £280.80, both including VAT, but with delivery likely on top. So roughly £500 for filters.

it’s also worth noting that this unit has an air fitting to provide for the air assist as well.

Counting against this unit is its age and how long the filters are likely to be available in the future.

**Comment by johnmckerrell** [comment]
It seems like this is closer to the AD Access which we determined was probably too small really anyway, and with having to get new filters wouldn't really be all that much cheaper (well, I guess the VAT makes it a bigger difference). So my feeling is it's still worth buying the AD Oracle new.

**Comment by JackiePease** [comment]
HPC are sending a proforma for the Oracle, so that we can pay by bank transfer tomorrow.

**Comment by JackiePease** [comment]
The AD Oracle has arrived and is currently by the Bookcase of Things waiting to be unwrapped. 
We need to check it has been delivered intact etc. but bear in mind that it will probably first be used in our temporary accessible space.
We might also need a different reducer.

**Comment by JackiePease** [comment]
We got this and used it for around 2 years with laser cutter Sheila - hopefully nobody has any objections to closing the issue.

It's currently in the laser room, but not in use due to the cost of replacement filters.
